src/main/web/.well-known/security.txt

```diff
@@ -1,2 +1,4 @@
 Contact: https://hackerone.com/52fa7bc0-5356-4c86-9f79-eeb03e1d55cc/embedded_submissions/new
-Policy: https://www.ons.gov.uk/help/vulnerabilitydisclosurepolicy
\ No newline at end of file
+
```

Comment:
The white space that has been added here is not in the develop branch. I am not sure if whatever uses this file is OK with the white space, but I think it would be safer to keep it as is?

Reply:
I don't think the whitespace should be there. I'll change it.